app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
Please remove express.json for now. This isn't required until you get to 'post' so you shouldn't need it at this point 🙂